src/netgroup.h

```diff
@@ -41,6 +41,16 @@ class NetGroupManager {
      */
     uint32_t GetMappedAS(const CNetAddr& address) const;
 
+    /**
+     *  Analyze and print current health of ASMap based buckets.
```

Comment:
nit:
```suggestion
     *  Analyze and log current health of ASMap based buckets.
```

Reply:
Thanks, I will change this when I have to retouch.

Reply:
done